org.eclipse.wildwebdeveloper.feature/feature.xml

```diff
@@ -2,7 +2,7 @@
 <feature
       id="org.eclipse.wildwebdeveloper.feature"
       label="%name"
-      version="0.7.0.qualifier"
+      version="0.7.1.qualifier"
```

Comment:
I didn't know what the next version should be, so I made the minimum change necessary for the build to complete.